engine/src/test/scala/engine/workflow/ValidateActorSpec.scala

```diff
@@ -59,22 +67,22 @@ class ValidateActorSpec extends TestKit(ActorSystem("ValidateActorSpecSystem"))
   "A Validate actor" should {
     "return ValidationSuccess when Namespace, options, runtime attributes and inputs are valid" in {
       within(Timeout) {
-        validateActor ! ValidateWorkflow(validWdlSource, Some(validInputsJson), Some(validOptionsJson))
+        validateActor ! ValidateWorkflow(validWdlSource, Some(validInputsJson), Some(customizedLocalBackendOptions))
         expectMsgPF() {
           case validationSuccess: ValidationSuccess =>
             if (validationSuccess.namespaceWithWorkflow.tasks.size != 1) fail("Number of tasks is not equals to one.")
             if (!validationSuccess.coercedInputs.get.head._1.contains("hello.hello.addressee")) fail("Input does not contains 'hello.hello.addressee'.")
-            if (!validationSuccess.workflowOptions.get.get("hello.hello.addressee").get.contains("world")) fail("Workflow option does not comply with 'hello.hello.addressee' entry.")
+            if (!validationSuccess.workflowOptions.get.get("backend").get.contains("retryableCallsSpecBackend")) fail("Workflow option does not comply with 'backend' entry.")
             if (!validationSuccess.runtimeAttributes.head.head.contains("docker")) fail("Runtime attribute does not contains 'docker' entry.")
           case unknown => fail(s"Response is not equals to the expected one. Response: $unknown")
         }
       }
-      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(validOptionsJson).get)
+      verify(backendMock, times(1)).assertWorkflowOptions(WorkflowOptions.fromJsonString(customizedLocalBackendOptions).get)
```

Comment:
Are these verify(..) statements are useful anymore, since it is using backendMock to verify the details (while the validation initializes a backend each time a `ValidateWorkflow` msg is received?